### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by iberflow** [droath/RecurlyClient#2]
Hi there,

As I can see this package is no longer maintained, but I was wondering whether you'd accept a pull request or maybe it's possible if I could take over?

I've upgraded the package dependencies to guzzle 6.1 and serializer 3.0, but the tests are failing hard so I'll try to sort it out this week.

**Comment by droath** [comment]
@ignasbernotas Yes, I'll accept a pull request. I've been neglecting maintaining this project but would like to get started again. Please feel free to post any issues or feature request in the issue queue.
